src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesDomainEventListener.kt

```diff
@@ -33,7 +36,7 @@ class ActivitiesDomainEventListener(
     val sqsMessage: SQSMessage = objectMapper.readValue(message)
     when (sqsMessage.Type) {
       "Notification" -> {
-        val (eventType) = objectMapper.readValue<HMPPSDomainEvent>(sqsMessage.Message)
+        val (eventType) = objectMapper.readValue<HmppsDomainEvent>(sqsMessage.Message)
```

Comment:
This was previously pointing at an Incentives domain event which has a different shape. All we need here is the event type and additional information could be different on each message anyway.